dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1838,4 +1838,19 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
         @ApiParam(value = "GitHub installation ID", required = true) @FormParam("installationId") String installationId) {
         return githubWebhookRelease(repository, username, gitReference, installationId);
     }
+
+    @POST
+    @Path("/github/delete")
```

Comment:
Should I also change the endpoint for releases? Currently, it is POST /workflows/github/release. Does POST /workflows/github make more sense?

Reply:
1. On further reflection, I think repository and reference could just be path parameters instead of query parameters, although I'm not sure because the resource represented by this endpoint, a GitHub repo, is modifying a Dockstore workflow, which is a different resource. But maybe something like: `/workflows/github/<repository>/<reference>`. I don't know you could argue this stuff for days, so either way. That said, regardless of the path, I think it should be a DELETE.
2. I think POST is fine for the other endpoint, as the creates/modifies.

Reply:
Yeah the disconnect between resources made me a bit confused with how to name the path. I think I'll just use query params and the DELETE type.